MDANSE/Src/MDANSE/Framework/Jobs/McStasVirtualInstrument.py

```diff
@@ -141,23 +141,23 @@ def initialize(self):
         self._mcStasPhysicalParameters = {"density": 0.0}
         self._mcStasPhysicalParameters["V_rho"] = 0.0
         self._mcStasPhysicalParameters["weight"] = sum(
-            [self.trajectory.get_atom_property(s, "atomic_weight") for s in symbols]
+            self.trajectory.get_atom_property(s, "atomic_weight") for s in symbols
         )
         self._mcStasPhysicalParameters["sigma_abs"] = (
             np.mean(
-                [self.trajectory.get_atom_property(s, "xs_absorption") for s in symbols]
+                self.trajectory.get_atom_property(s, "xs_absorption") for s in symbols
```

Comment:
This probably won't work. `np.mean` requires an ArrayLike rather than geenrators.